var/spack/repos/builtin/packages/openspeedshop-utils/package.py

```diff
@@ -96,7 +96,7 @@ class OpenspeedshopUtils(CMakePackage):
     depends_on("dyninst@develop", when='@develop')
     depends_on("dyninst@10:", when='@2.3.1.3:9999')
 
-    depends_on("python", when='@develop', type=('build', 'run'))
+    depends_on("python@2.7.14:2.7.99", when='@develop', type=('build', 'run'))
     depends_on("python@2.7.14:2.7.99", when='@2.3.1.3:9999', type=('build', 'run'))
```

Comment:
Why not combine all of these by removing the `when` clause altogether?

Reply:
@adamjstewart I fixed this and submitted the changes.